2021-02-13 19:56:59,499 INFO    MainThread:8787 [wandb_setup.py:_flush():70] setting env: {}
2021-02-13 19:56:59,499 INFO    MainThread:8787 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-02-13 19:56:59,499 INFO    MainThread:8787 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-13 19:56:59,499 INFO    MainThread:8787 [wandb_init.py:_log_setup():319] Logging user logs to /home/ubuntu/payload/scripter/wandb/run-20210213_195659-3ge64rew/logs/debug.log
2021-02-13 19:56:59,499 INFO    MainThread:8787 [wandb_init.py:_log_setup():320] Logging internal logs to /home/ubuntu/payload/scripter/wandb/run-20210213_195659-3ge64rew/logs/debug-internal.log
2021-02-13 19:56:59,499 INFO    MainThread:8787 [wandb_init.py:init():324] calling init triggers
2021-02-13 19:56:59,499 INFO    MainThread:8787 [wandb_init.py:init():331] wandb.init called with sweep_config: {}
config: {}
2021-02-13 19:56:59,499 INFO    MainThread:8787 [wandb_init.py:init():401] starting backend
2021-02-13 19:56:59,512 INFO    MainThread:8787 [backend.py:ensure_launched():81] starting backend process...
2021-02-13 19:56:59,521 INFO    MainThread:8787 [backend.py:ensure_launched():86] started backend process with pid: 8892
2021-02-13 19:56:59,522 INFO    MainThread:8787 [wandb_init.py:init():410] backend started and connected
2021-02-13 19:56:59,524 INFO    MainThread:8787 [wandb_init.py:init():433] updated telemetry
2021-02-13 19:56:59,525 INFO    MainThread:8787 [wandb_init.py:init():456] communicating current version
2021-02-13 19:57:00,173 INFO    MainThread:8787 [wandb_init.py:init():461] got version response 
2021-02-13 19:57:00,173 INFO    MainThread:8787 [wandb_init.py:init():469] communicating run to backend with 30 second timeout
2021-02-13 19:57:00,281 INFO    MainThread:8787 [wandb_init.py:init():492] starting run threads in backend
2021-02-13 19:57:01,826 INFO    MainThread:8787 [wandb_run.py:_console_start():1411] atexit reg
2021-02-13 19:57:01,826 INFO    MainThread:8787 [wandb_run.py:_redirect():1274] redirect: SettingsConsole.REDIRECT
2021-02-13 19:57:01,827 INFO    MainThread:8787 [wandb_run.py:_redirect():1279] Redirecting console.
2021-02-13 19:57:01,827 INFO    MainThread:8787 [redirect.py:install():213] install start
2021-02-13 19:57:01,827 INFO    MainThread:8787 [redirect.py:install():228] install stop
2021-02-13 19:57:01,827 INFO    MainThread:8787 [redirect.py:install():213] install start
2021-02-13 19:57:01,827 INFO    MainThread:8787 [redirect.py:install():228] install stop
2021-02-13 19:57:01,828 INFO    MainThread:8787 [wandb_run.py:_redirect():1325] Redirects installed.
2021-02-13 19:57:01,828 INFO    MainThread:8787 [wandb_init.py:init():515] run started, returning control to user process
2021-02-13 19:57:02,090 INFO    MainThread:8787 [wandb_run.py:_atexit_cleanup():1381] got exitcode: 1
2021-02-13 19:57:02,091 INFO    MainThread:8787 [wandb_run.py:_restore():1353] restore
2021-02-13 19:57:02,091 INFO    MainThread:8787 [redirect.py:uninstall():232] uninstall start
2021-02-13 19:57:02,091 INFO    MainThread:8787 [redirect.py:_stop():287] _stop: stdout
2021-02-13 19:57:02,092 INFO    MainThread:8787 [redirect.py:_stop():293] _stop closed: stdout
2021-02-13 19:57:02,092 INFO    stdout    :8787 [redirect.py:_pipe_relay():129] relay done saw last write: stdout
2021-02-13 19:57:02,092 INFO    stdout    :8787 [redirect.py:_pipe_relay():145] relay done done: stdout
2021-02-13 19:57:02,092 INFO    MainThread:8787 [redirect.py:_stop():299] _stop joined: stdout
2021-02-13 19:57:02,092 INFO    MainThread:8787 [redirect.py:_stop():301] _stop rd closed: stdout
2021-02-13 19:57:02,092 INFO    MainThread:8787 [redirect.py:uninstall():236] uninstall done
2021-02-13 19:57:02,092 INFO    MainThread:8787 [redirect.py:uninstall():232] uninstall start
2021-02-13 19:57:02,093 INFO    MainThread:8787 [redirect.py:_stop():287] _stop: stderr
2021-02-13 19:57:02,093 INFO    MainThread:8787 [redirect.py:_stop():293] _stop closed: stderr
2021-02-13 19:57:02,093 INFO    stderr    :8787 [redirect.py:_pipe_relay():129] relay done saw last write: stderr
2021-02-13 19:57:02,093 INFO    stderr    :8787 [redirect.py:_pipe_relay():145] relay done done: stderr
2021-02-13 19:57:02,093 INFO    MainThread:8787 [redirect.py:_stop():299] _stop joined: stderr
2021-02-13 19:57:02,093 INFO    MainThread:8787 [redirect.py:_stop():301] _stop rd closed: stderr
2021-02-13 19:57:02,093 INFO    MainThread:8787 [redirect.py:uninstall():236] uninstall done
2021-02-13 19:57:02,186 INFO    MainThread:8787 [wandb_run.py:_wait_for_finish():1504] got exit ret: file_counts {
  wandb_count: 2
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 6253
  total_bytes: 6253
}

2021-02-13 19:57:04,190 INFO    MainThread:8787 [wandb_run.py:_wait_for_finish():1504] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 11438
  total_bytes: 11438
}

2021-02-13 19:57:05,455 INFO    MainThread:8787 [wandb_run.py:_show_files():1726] logging synced files
